Changelogs for Digital Twin Manager Microservice 1024.1.0 (#4235)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1024-1-0-changed-the-structure-of-storing-ordering-information-of-complex-property-definitions-existing-property-definitions-are-migrated-to-the-new-structure.md

---
date: ""

> title: "Changed the structure of storing ordering information of complex Property Definitions; existing Property Definitions are migrated to the new structure"

Reviewer: ```suggestion
title: "Changed the storage of ordering information of complex property definitions
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-2c7RdTdXo4"
      label: "Improvement"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1918"
version: "1024.1.0"
---

> Previously the field ordering information of complex Property Definition

Reviewer: ```suggestion
Previously, the field ordering information of complex property definitions
```

> was stored in the `c8y_Order` array. This changed now and the order is

Reviewer: ```suggestion
was stored in the `c8y_Order` array. With this change, the order is
```

stored in the `order` property within each field of the JSON schema.

> Existing Property Definitions are migrated to the new structure when the

Reviewer: ```suggestion
Existing property definitions are migrated to the new structure when the
```
